src/components/chimera.svg

```diff
@@ -0,0 +1,13 @@
+<svg height="511pt" viewBox="-8 0 511 511.96139" width="511pt" xmlns="http://www.w3.org/2000/svg" style="fill: #f3cfd6;">
```

Comment:
good old `pt`, `px` might be more appropriate 🎓

Reply:
fixed 🙇‍♀